tests/testapps/CTCatalog/CTCatalog/Tests/CTCFontTestViewController.mm

```diff
@@ -91,6 +91,13 @@ - (void)drawRect:(CGRect)rect {
     // Draws the text in the frame
     CTFrameDraw(frame, context);
 
+    UniChar characters[6];
+    [@"Glyphs" getCharacters:characters range:NSMakeRange(0, 6)];
```

Comment:
yeah this seems unnecessary, I'm pretty sure there's a way you can write it as {'G', 'l', 'y', 'p', 'h', 's'}

---
In reply to: [91414988](https://github.com/Microsoft/WinObjC/pull/1516#discussion_r91414988) [](ancestors = 91414988)